### PR TITLE
Admin UI: replaced ContainerQuery component with useResizeObserver

### PR DESCRIPTION
**Pull request by Vultraz**
*sweep sweep sweep*

### CODE REVIEW

**Comment by MadeByMike** [comment]
🤔 I don't even want to own this code. Can we just add something like: https://github.com/ZeeCoder/use-resize-observer

**Comment by Vultraz** [comment]
@MadeByMike ah, good idea. Done!

**Comment by Vultraz** [comment]
Agh, dammit, now that `Uncaught Error: ResizeObserver loop limit exceeded` error shows up again. >_<

File: packages/app-admin-ui/client/pages/Home/index.js [hunk 1]
```diff
@@ -56,6 +24,17 @@ const HomepageListProvider = () => {
   const query = gqlCountQueries(lists);
   const { data, error } = useQuery(query, { fetchPolicy: 'cache-and-network', errorPolicy: 'all' });
 
+  const [cellWidth, setCellWidth] = useState(3);
+  const { ref: measureElement } = useResizeObserver({
+    onResize: ({ width }) => {
+      // requestAnimationFrame works around a weird 'ResizeObserver loop limit exceeded' error.
```
Comment (MadeByMike) [inline packages/app-admin-ui/client/pages/Home/index.js]: `use-resize-observer` has a throttle\debounce method for this: https://github.com/ZeeCoder/use-resize-observer#throttle--debounce

I think we want to throttle it to about 15 fps ie around 60ms

Comment (Vultraz) [reply packages/app-admin-ui/client/pages/Home/index.js]: Switched to using `lodash.throttle` as the docs suggest.

Comment (Vultraz) [reply packages/app-admin-ui/client/pages/Home/index.js]: Looks like `requestAnimationFrame` was still required after all.

Comment (Vultraz) [reply packages/app-admin-ui/client/pages/Home/index.js]: Ok, `throttle` doesn't fix the problem (see https://github.com/WICG/ResizeObserver/issues/38). `requestAnimationFrame` was needed after all.

**Comment by timleslie** [comment]
@Vultraz Could you fix up the merge conflict?
@MadeByMike Could you re-review this and give it a ✅ if you're happy with it?

**Comment by Vultraz** [comment]
@timleslie done

**Comment by MadeByMike** [comment]
Nice! Thanks @Vultraz

**Comment by timleslie** [comment]
I'm trying to work out why these tests are failing. In the meantime I'm gonna keep re-triggering until this goes green :-) If nothing else it will give me insight into the tests and how and why they might be failing!

**Pull request merged**